src/string-calculator.ts

```diff
@@ -169,4 +169,19 @@ export class StringCalculator{
 
         return numberList.indexOf("\n\n")+1;
     }
+
+    private getUnexpectedChar(number:string) :string {
+        let chars:string[] = number.split("");
```

Comment:
this method does not look good. you can strip all digits in this string and be left with non-digits only. since you return the first found non-digit, you either expect it to be only one or you can return first from the result